internal/campaigns/executor.go

```diff
@@ -312,18 +313,18 @@ func createChangesetSpec(task *Task, diff string) *ChangesetSpec {
 			BaseRef:        task.Repository.BaseRef(),
 			BaseRev:        task.Repository.Rev(),
 			HeadRepository: task.Repository.ID,
-			HeadRef:        "refs/heads/" + task.Template.Branch,
-			Title:          task.Template.Title,
-			Body:           task.Template.Body,
+			HeadRef:        "refs/heads/" + task.Template.Branch.Value(repo),
+			Title:          task.Template.Title.Value(repo),
+			Body:           task.Template.Body.Value(repo),
 			Commits: []GitCommitDescription{
 				{
-					Message:     task.Template.Commit.Message,
+					Message:     task.Template.Commit.Message.Value(repo),
 					AuthorName:  authorName,
 					AuthorEmail: authorEmail,
 					Diff:        string(diff),
 				},
 			},
-			Published: task.Template.Published,
+			Published: task.Template.Published.Is(repo),
```

Comment:
Tiniest nitpicky thought: after reading `Value` and then `Is` I'm wondering whether a generic method name that says "give me the value for this repo" makes sense to have for every field that's overridable, like `ValueFor(repo)` for example.